acceptance-tests/src/test/java/linea/plugin/acc/test/TransactionTraceLimitOverflowTest.java

```diff
@@ -68,7 +68,7 @@ public void transactionOverModuleLineCountRemoved() throws Exception {
             GAS_PRICE.multiply(BigInteger.TEN).add(BigInteger.ONE));
     final byte[] signedTxContractInteraction =
         TransactionEncoder.signMessage(
-            txModuleLineCountTooBig, Credentials.create(Accounts.GENESIS_ACCOUNT_TWO_PRIVATE_KEY));
+            txModuleLineCountTooBig, Credentials.create(Accounts.GENESIS_ACCOUNT_ONE_PRIVATE_KEY));
```

Comment:
Can you please explain how this change is related to `sendBundle` and the bug we had?

Reply:
the intent of this test is to verify that, if during block building, there is a tx that goes over the limits then following valid txs are included in the block, but it reality before valid txs were evaluated before the invalid one, due the gas price, and this specific line is changed because the sender of the invalid tx should not be the sender of the valid txs, otherwise the invalid tx will be replaced.